Samples~/ThirdPersonDemoURP/Demo/Scripts/Player/PlayerInputControl.cs

```diff
@@ -12,17 +13,23 @@ public class PlayerInputControl : MonoBehaviour
         [Header("Movement Settings")]
         public bool analogMovement;
 
+        [Header("Bot Settings")] public bool isBot = true;
+
         private void Update()
         {
-            MoveInput(new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical")));
-            JumpInput(Input.GetButtonDown("Jump"));
+            if (!isBot)
+            {
+                MoveInput(new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical")));
+                JumpInput(Input.GetButtonDown("Jump"));
+            }
         }
 
+        [RGAction("MoveInDirection")]
         public void MoveInput(Vector2 newMoveDirection)
         {
             move = newMoveDirection;
         }
-
+        
```

Comment:
Just a note that some of our IDEs add white space, which gets checked in, then others remove it, which gets checked in. Rider can share a `.DotSettings` file that would keep all of our IDEs in-sync for style and remove back-and-forth changes for things like new-lines, whitespace, etc

https://blog.jetbrains.com/dotnet/2017/02/20/rider-eap-update-code-style-settings-layers/